<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="LoginTextEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Логин</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordTextEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Пароль</string>
   </property>
   <property name="cursorMoveStyle">
    <enum>Qt::LogicalMoveStyle</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="SignUpButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Регистрация</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="OkButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>ОК</string>
   </property>
  </widget>
  <widget class="QLabel" name="InfLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
